stdlib/public/CommandLineSupport/CommandLine.cpp

```diff
@@ -244,7 +203,16 @@ static void swift::enumerateUnsafeArgv(const F& body) {
   int argc = 0;
   if (LPWSTR *wargv = CommandLineToArgvW(GetCommandLineW(), &argc)) {
     std::for_each(wargv, wargv + argc, [=] (wchar_t *warg) {
-      auto arg = copyUTF8FromWide(warg);
+      auto arg = swift::win32::copyUTF8FromWide(warg);
```

Comment:
Suggestion:

```c++
if (auto arg = swift::win32::copyUTF8FromWide(warg)) {
  body(argc, arg);
  free(arg);
} else {
  // Note that GetLastError() and errno may not be so useful here,
  // as in the error case we may have called free(), which might reset
  // either or both of them.
  swift::fatalError(0,
                      "Fatal error: Unable to convert argument '%ls' to "
                      "UTF-8: %lx, %d.\n",
                      warg, ::GetLastError(), errno);
}
```

Watch out for those 2-character tabs.

Reply:
:-) If you use Emacs, which I do, the Swift sources configure Emacs automatically to do appropriate indentation.